### PR TITLE
Added lp12 generators

### PR DESCRIPTION
**Pull request by stephanepericat**
Hello @operatino ,

As discussed this morning, I added templates to generate lp12 widgets and modules via our beloved cli :)
let me know what you think!

Cheers,
Stéphane

### CODE REVIEW

File: templates/lp12-module/bbscaff.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            name: 'module_name',
+            message: 'Module name',
+            'default': function(answers) {
+                return answers.module_title.replace(/ /gi, '-').toLowerCase();
+            }
+        },
+        {
+            name: 'module_author',
+            message: 'Module author',
+            'default': 'Backbase <practices@backbase.com>'
```
Comment (operatino) [inline templates/lp12-module/bbscaff.js]: I think it's better to leave it blank by default.

File: templates/lp12-module/bbscaff.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        },
+        {
+            name: 'module_version',
+            message: 'Module version',
+            'default': '1.0.0'
```
Comment (operatino) [inline templates/lp12-module/bbscaff.js]: Following semantic versioning, it's better to start with `0.1.0`. Unless it's political question.

File: templates/lp12-module/template/.gitignore [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+web_modules
+bower_components
+resources
+features
+docs
+vendors
+sandbox
+pids
+logs
+results
```
Comment (operatino) [inline templates/lp12-module/template/.gitignore]: Is this list ignored files list pre-defined by LP? `results`, `features`, `resources` and etc looks very suspicious.

File: templates/lp12-module/template/scripts/main.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
+/**
+ *  ----------------------------------------------------------------
+ *  Copyright © Backbase B.V.
+ *  ----------------------------------------------------------------
+ *  Author : Backbase R&D - Amsterdam - New York
```
Comment (operatino) [inline templates/lp12-module/template/scripts/main.js]: In this case Author is most likely not Backbase R&D.

File: templates/lp12-widget/template/README.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+- List of event the widget publishes/subscribes
+
+## Custom Components
+- list of widget custom components (if any)
+
+## Develop Standalone
+
+<!-- ```bash
+git clone <repo-url> && cd <widget-path>
+bb install -l
```
Comment (operatino) [inline templates/lp12-widget/template/README.md]: There's no `bb install`, I see it's messed-up with `bblp`. Also, I would expect to have some info about installing CLI tools first.

File: README.md [hunk 6]
```diff
@@ -56,6 +56,18 @@ $ bb generate widget
 $ bb generate container
 ```
 
+#### launchpad 12 Generators
```
Comment (operatino) [inline README.md]: Uppercase please - "Launchpad 0.12 Generators"

File: templates/lp12-module/template/.gitignore (hunk 3 above)
Comment (stephanepericat) [inline templates/lp12-module/template/.gitignore]: yes it is part of the lp12 widget archetype... i know, looks weird.

**Pull request merged**